scss/_variables.scss

```diff
@@ -1896,7 +1904,7 @@ $step-item-bg:           $black !default;
 $step-item-active-bg:    $primary !default;
 $step-item-next-bg:      $gray-400 !default;
 $step-item-shadow-size:  $border-width * 1.5 !default;
-$step-item-drop-shadow:  drop-shadow($step-item-shadow-size 0 0 $white) #{"/* rtl:"} drop-shadow(-$step-item-shadow-size 0 0 $white) #{"*/"} !default;
+$step-item-drop-shadow:  drop-shadow($step-item-shadow-size 0 0 $white) #{"/* rtl:"} drop-shadow(calc($step-item-shadow-size * -1) 0 0 $white) #{"*/"} !default;  /* stylelint-disable-line function-disallowed-list */
```

Comment:
Ping to remove calc ?

Reply:
Removed via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1167/commits/ac136b9242e15b88589b0f0e18cffabf4b4de85a